imaids/blocks.py

```diff
@@ -349,3 +362,8 @@ def bounding_box_center(self):
         center = _np.mean(self.bounding_box(), axis=1)
         
         return list(center)
+
+    def _check_magnetization(self, magnetization):
+        if len(magnetization) != 3:
+            raise ValueError('Invalid magnetization argument.')
+        return magnetization
```

Comment:
there is no need to return magnetization here, only a check would suffice. Maybe we could consider returning `True` or `False` instead of `ValueError`.

Reply:
hi @lucashfrancisco , what do you think about this?

Reply:
@fernando, I return magnetization here so that __init__ could both check its value and receive it after checking before settings its _magnetization attribute.

Reply:
But why is that useful? Wouldn't be more pythonic to not return anything or return a boolean indicating if the value is ok?

Reply:
it is useful because this checking has to be done in `__init__` and also in `magnetization` setter. By returning the magnetization we can condense the vector checking and the attribute `_magnetization` in one line. Returning a boolean and processing it in these two methods effectively is replicating code which could be only in `_check_magnetization` into the two calling functions. For example:

```
__init__:
     status =  self._check_magnetization(magnetization)
     if not status:
         raise ValueError('Invalid magnetization argument.')
      self._magnetization = self._check_magnetization(magnetization)
  
@magnetization.setter:
     status =  self._check_magnetization(magnetization)
     if not status:
         raise ValueError('Invalid magnetization argument.')
     self._magnetization = self._check_magentization(magnetization)
```

compared to 

```
__init__:
      self._magnetization = self._check_magentization(magnetization)
  
@magnetization.setter:
     self._magnetization = self._check_magentization(magnetization)
```

Reply:
Well... I disagree with your first implementation... actually it would be something like this:
```
__init__:
      self.magnetization = magnetization
  
@magnetization.setter:
     if len(magnetization) != 3:
         raise ValueError('Invalid magnetization argument.')
     self._magnetization = magnetization
```
It seems that even the existence of the method `_check_magnetization` is not needed in this case and we would have the advantage of making use of the setter appropriately...
What do you think?

Reply:
oh, I see, @fernandohds564 . Now, I understand your proposal. I did not see your last message before talking to you in person moments agor. I agree. I will do it.